app/models/order_import.rb

```diff
@@ -131,8 +131,16 @@ def handle_save_nothing_on_error # TODO refactor and rename
   end
 
   def import_row(row) # TODO refactor
-    row_importer = OrderRowImporter.new(row, self)
-    row_importer.import
+    begin
+      row_importer = OrderRowImporter.new(row, self)
+      row_importer.import
+    rescue => e
```

Comment:
No, it won't send emails for invalid headers since we're validating those now and including them in the error report.

This email is sent in the case the order row importer throws an exception. I figured we want that to happen instead of letting it fail silently with delayed_job.